### DISCUSSION TITLE
Res18 & Res34

### DISCUSSION THREAD

**Discussion by DjangoChaogh** [STVIR/pysot#323] [General]
I notice that authors modified the default structure of backbones, e.g. ResNet-50, ResNet-18 and moblienetV2. These modifications are significant to SiamRPN++. Though authors published the backbone of ResNet-50., my question is, how could I train the SiamRPN++ with the modified Res18 or Res34? Could you upload these version of backbones? or where should I download them?

![65814495-31405100-e215-11e9-84c2-009cbcf96c33](https://user-images.githubusercontent.com/42630571/76697813-d91fdd00-66d6-11ea-9c70-7fc7a987bc3f.png)
![Uploading 65814556-ed9a1700-e215-11e9-9060-9b02eb3aee54.png…]()

**Comment by ZhiyuanChen** [comment]
The reason why the authors modified the last 2 blocks is that the default stride (32) is tooooo large for object tracking. So just change the stride accordingly should be fine.

**Comment by ZhiyuanChen** [comment]
Since no further quesion is asked, I'm closing it for now. 
Please open a new issue should you have more questions.
